site-container.yml.sample

```diff
@@ -57,7 +57,7 @@
     - import_role:
         name: ceph-container-common
       tags: fetch_container_image
-      when: not (inventory_hostname in groups.get('clients', [])) or (inventory_hostname == groups.get('clients', [''])|first)
+      when: (group_names != ['clients']) or (inventory_hostname == groups.get('clients', [''])|first)
```

Comment:
I'm just concerned about the fact people could use groups of groups
eg:

let's you have this inventory:

```ini
[clients:children]
client_osp
client_other

[client_osp]
client_osp0
client_osp1
client_osp2

[client_other]
client_other0
client_other1
client_other2
```

for instance, `client_osp0` will have `group_names = ['client_osp', 'clients']`

eg:

running this play:
```yaml
- hosts: client_osp0
  become: true
  gather_facts: True
  tasks:
    - name: print '{{ group_names }}'
      debug:
        msg: "{{ group_names }}"
```
output:

```
PLAY [client_osp0] ***********************************************************************************************************************************************************************************************************************************************************************************************************

TASK [Gathering Facts] *******************************************************************************************************************************************************************************************************************************************************************************************************
ok: [client_osp0]

TASK [print '['client_osp', 'clients']'] *************************************************************************************************************************************************************************************************************************************************************************************
ok: [client_osp0] => 
  msg:
  - client_osp
  - clients

PLAY RECAP *******************************************************************************************************************************************************************************************************************************************************************************************************************
client_osp0                : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
```